pkg/helm/helm_test.go

```diff
@@ -79,6 +79,11 @@ func TestHelmCache(t *testing.T) {
 }
 
 func TestHelmRegistry(t *testing.T) {
+	// Test should only run as integration test if registry is available
+	if host, isSet := os.LookupEnv("DOCKER_HOSTNAME"); !isSet || host != "localhost:5000" {
```

Comment:
Not sure about this. Theoretically, I can run integration tests from any environment.

Reply:
True. So how about dropping the host check and only skipping the test if the `DOCKER_HOSTNAME` environment variable is not set?

Reply:
Agree.